Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m

```diff
@@ -1172,6 +1185,59 @@ - (void)playMediaFile{
         
         [self.avMoviePlayer.contentOverlayView addObserver:self forKeyPath:[MediaAVPlayerViewController observerKeyFullScreen] options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld context:NULL];
         
+
+        if (image != nil) {
+            UIImageView *imageView = [[UIImageView alloc] initWithFrame:CGRectMake(0,0,300,300)];
```

Comment:
Is the width and height a static size?

Reply:
Yep, but it is auto resized and centered based on NSLayoutConstraints, any idea how to initialize without the hardcoded sizes?